### PR TITLE
2023.08.07

### PR DESCRIPTION
**Pull request by UTsweetyfish**
* remove panda related code.
  * fix sed logic on journald.conf.
    issue: https://github.com/linuxdeepin/developer-center/issues/5127
  * d/control: add python3-gi dependency for dde-fist-run.

### CODE REVIEW

**Review by BLumia — DISMISSED** [review]
竟然之前还留着15专业版的codename并且还没出问题- -

File: Makefile [hunk 1]
```diff
@@ -1,24 +1,11 @@
-ARCH=$(shell dpkg --print-architecture)
-SYSTYPE=Desktop
-SYSTYPE=$(shell cat /etc/deepin-version | grep Type= | awk -F'=' '{print $$2}')
-ifeq ($(ARCH)_$(SYSTYPE), amd64_Desktop)
-	ORIGIN=false
-else ifeq ($(ARCH)_$(SYSTYPE), i386_Desktop)
-	ORIGIN=false
-else
-	ORIGIN=false
-endif
```
Comment (UTsweetyfish) [inline Makefile:10]: > 竟然之前还留着15专业版的codename并且还没出问题- -

因为这里都是 false

**Review by BLumia — APPROVED** [review]

**Pull request merged**